dts/msm8916/msm8916-mtp-smb1360.dts

```diff
@@ -9,6 +9,24 @@
 	qcom,msm-id = <206 0>;
 	qcom,board-id = <8 1>;
 
+	/*
+	* To get a working build for acer-a1-724, comment out
+	* $(LOCAL_DIR)/msm8916-mtp-smb1360.dtb in rules.mk in this directory.
```

Comment:
Did you mean to comment out msm8916-mtp.dts?

Reply:
Not really. I mean "comment out all dtbs and add $(LOCAL_DIR)/msm8916-mtp-smb1360.dtb".

Reply:
comment out **all lines except** ...

maybe then?